### DISCUSSION TITLE
Is there a way to format indicators i.e. number of digits like {load: 8.2} ??

### DISCUSSION THREAD

**Discussion by fooryo** [Alexays/Waybar#3161] [Q&A]
I would like to have some numerical (float) indicator to be visible with a fixed amount of digits or precision.

Something like {load: 8.2f} for the CPU load indicator showing me float with 8 char wide,2 digits after the point, filled with spaces.

Is it possible? I didn't manage to do that.

**Comment by Jukelyn** [comment]
So looking at the CPU code and trying some things out, the `{load:8.2f}` does work to have a width of 8 with 2 decimal precision. As for the others, those do not support float formatting as far as I can tell. From my testing this is what I found:

| string | type (kind of formatting) |
| ------ | -----------|
| load | float (f) |
| usage | integer (d) |
| frequency | unsigned long (same as string?) |

I'm not too familiar with cpp and so i'm not sure if the *types* are correct here but the type of formatting that I used seems to work for those.

I did these to test:
`{load:0.1f}`, `{load:0.2f}`, `{load:<6.2f}`, and `{load:>6.2f}`
`{usage:12d}`, `{usage:03d}`, `{usage:>04d}`, and `{usage:<04d}`
`{avg_frequency:>6}`, `{avg_frequency:<6}`, and `{avg_frequency:^6}`

I don't use icons so I couldn't test those.
